### ISSUE TITLE
Support Json API for convenient data parsing

### ISSUE THREAD

**Issue by ee07b415** [ordinals/ord#2010]
Do we have some plan to support some API return the Json object of the inscription/satoshi? The current API return the whole pageHtml is hard to parse the data and I can imagine that the template for the html could change more frequent than the data obj.

**Comment by raphjaph** [comment]
We've started work on a JSON API and will add more endpoints in the coming weeks: #2250
